content/SCALE/SCALETutorials/Shares/NVMe-oF/_index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "NVMe-oF Subsystems"
+description: "Provides information on NVMe-oF subsystems and instruction on creating a subsystem and setting up enterprise configurations of subsystems."
+geekdocCollapseSection: true
+weight: 90
+related: false
+aliases:
+ -
+tags:
+- nvme-of
+- nvme
+- nvme baseNQN
+- block shares
+- subsystems
+- namespaces
+- hosts
+- ports
+- rdma
+- ana
+- asymmetric namespace access
+- remote direct memory access
+- cross-port referrals
+---
+
+## Overview of NVMe-oF
+
+{{< include file="/static/inclues/NVMe-oF-Overview.md" >}}
+
+## Configuring the NVMe-oF Service
+
+You can access the NVMe-of service screen from the:
+* <span class="material-icons">more_vert</span> dropdown menu on the **NVMe-oF Subsystems** widget on the **Shares** screen
+* **Global Configuration** button at the top of the **NVMe-oF** screen
+* **NVMe-oF** service option on the **System > Services** screen
+
+The **NVMe-oF Global Configuration** shows the base NQN for the service.
+
+{{< expand "What is the base NQN?" "v" >}}
+The *base NQN* us the standardized NVMe Qualified Name for the service.
```

Comment:
1. "Base NQN"  (i.e. upper case B)
2. It's the **prefix** that will be used when **new** subsystems are created.  Oh, I see now you mention that below.  Changing it will **not** change previously configured subsystem NQNs (subnqns).  

(FYI, the reason, is that we might have configured the client to talk to a particular NQN.  We therefore don't want to change them too easily, breaking the client connection.)

Maybe get another opinion wrt this section.  I think it may need a tidy-up.

Reply:
I've reworked this a bit in the next commit.